Rethinking_2/Chp_02.ipynb

```diff
@@ -0,0 +1,409 @@
+{
```

Comment:
<ul><li>I think we should use <a href="https://github.com/rasbt/watermark" target="_blank">the watermark package</a> for package info and add it to the environment requirements file, as I did for your BAP repo.</li><li>We should also add black formatter and a package that automatically blackifies notebooks. I use <a href="https://github.com/dnanhkhoa/nb_black" target="_blank">nb_black</a> but I noticed it messes up conda envs recently. Do you know another one?</li></ul><p> </p><p>What do you think?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I have the autopep8 extension (I think from here https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/install.html). I have never use black with jupyter notebooks. What about https://github.com/drillan/jupyter-black.</p><p>Also I do not really like how blacks deals with parentheses, I see the benefit but I do not like how it looks, haha.</p>

Reply:
<p>Ha ha I see. I like it, but I'd say it's your call, based on seniority! Or Ravin can break this tie for us ;)</p><p>In any case, before settling this issue, do you agree on adding watermark to environment.yml? Also, shouldn't we add pandas and numpy?</p>